packages/appeals-service-api/src/repositories/sql/appeal-user-repository.js

```diff
@@ -87,6 +87,9 @@ class AppealUserRepository {
 		return await this.dbClient.appealUser.findUnique({
 			where: {
 				email
+			},
+			include: {
```

Comment:
I think this would be better done in a separate api call/repo function (or base off of a param)
Majority of users aren't rule 6 and the majority of calls to this won't need this data and this is probably a hot path, so best to avoid repeat lookups to this table

Reply:
If interested you could look at using a scope request for the id token that returns if a user has rule6 status from the auth server. We can then check this on api requests without additional db lookups to secure any rule 6 api only endpoints

Reply:
I've added a new api call rather than using the scope request - can investigate that further if you think it's a better approach